Models/Vendedor.cs

```diff
@@ -16,29 +18,29 @@ public Vendedor()
         Cpf = "";
         CnpjEmpresa = "";
         Nome = "";
-        ListaVendas = [];
+        Vendas = [];
     }
         
     public decimal CalcularSalario()
     {
         
-            if (ListaVendas == null || ListaVendas.Count == 0 || PercentualComissao == 0)
+            if (Vendas == null || Vendas.Count == 0 || PercentualComissao == 0)
             {
                 return SalarioFixo;
             }
             if (PercentualComissao < 0)
             {
-                throw new Exception("Percentual de comissão não pode ser negativo");
+                throw new ComissaoNegativaException("Percentual de comissão não pode ser negativo");
             }
             if (SalarioFixo < 0)
             {
-                throw new Exception("Salário fixo não pode ser negativo");
+                throw new SalarioNegativoException("Salário fixo não pode ser negativo");
```

Comment:
Essa checagem ocorre após a checagem `Vendas == null || Vendas.Count == 0 || PercentualComissao == 0`, o que significa que se o salário fixo for negativo mas uma das condições acima ocorrer, será retornado um valor negativo em vez de lançada a exceção.